systemverilog-plugin/UhdmAst.cc

```diff
@@ -1758,17 +1747,15 @@ void UhdmAst::process_packed_array_typespec()
             current_node->str = str;
             delete node;
         } else if (node) {
-            current_node->str = node->str;
-            if (node->type == AST::AST_ENUM && !node->children.empty()) {
-                for (auto *c : node->children[0]->children) {
-                    if (c->type == AST::AST_RANGE && c->str.empty())
-                        packed_ranges.push_back(c);
-                    else
-                        delete c;
-                }
-                node->children[0]->children.clear();
+            if (!node->str.empty()) {
+                auto wiretype_node = make_ast_node(AST::AST_WIRETYPE);
+                wiretype_node->str = node->str;
+                current_node->children.push_back(wiretype_node);
```

Comment:
```suggestion
                AST::AstNode *const wiretype_node = make_named_node(AST::AST_WIRETYPE);
                wiretype_node->str = node->str;
                current_node->children.push_back(wiretype_node);
```
Let's use `make_named_node` instead of `make_ast_node` everywhere, with the intention of removing `make_ast_node` someday. They do the same thing, but the former returns `AstNodeBuilder`, which makes it more flexible.